### ISSUE TITLE
Loss does not decrease during training

### ISSUE THREAD

**Issue by vvgoder** [TowardsNorth/yolo_v1_tensorflow_guiyu#4]
loss decrease at the beginning of training ,but the loss remained at 6-10  whether i increase the training step or decrease the learning rate,please help me !
